releases/release-8.5.1.md

```diff
@@ -0,0 +1,100 @@
+---
+title: TiDB 8.5.1 Release Notes
+summary: 了解 TiDB 8.5.1 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 8.5.1 Release Notes
+
+发版日期：2025 年 x 月 x 日
+
+TiDB 版本：8.5.1
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v8.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v8.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v8.5.1#version-list)
+
+## 操作系统支持变更
+
+- 从 v8.5.1 起，TiDB 重新提供了对 CentOS Linux 7 的支持。如果你需要在 CentOS 7 上部署 TiDB 8.5 版本或将集群升级到 TiDB 8.5 版本，请部署或升级至 TiDB 8.5.1 或以上版本。
+    - 根据 [CentOS Linux EOL](https://www.redhat.com/en/blog/centos-linux-has-reached-its-end-life-eol)，CentOS Linux 7 的上游支持已于 2024 年 6 月 30 日终止。在 v8.4.0 DMR 和 v8.5.0 版本中，TiDB 暂停了对 CentOS 7 的支持，建议使用 Rocky Linux 9.1 及以上的版本。如果在使用 CentOS Linux 7 的情况下将 TiDB 升级到 v8.4.0 DMR 或 v8.5.0 版本，将导致集群不可用的风险。
+    - 尽管 TiDB v8.5.1 及以上版本支持 CentOS Linux 7，但由于 CentOS Linux 7 已经达到其生命周期的终止（EOL），强烈建议用户参考该系统的[官方声明和安全建议](https://www.redhat.com/en/blog/centos-linux-has-reached-its-end-life-eol)，考虑尽快迁移到 [TiDB 支持的操作系统版本](/hardware-and-software-requirements.md#操作系统及平台要求)，如 Rocky Linux 9.1 及以上版本。
+
+## 改进提升
+
++ TiDB <!--tw@Oreoxmt: 5 notes-->
+
+    - 支持将只读的用户自定义变量折叠为常量 [#52742](https://github.com/pingcap/tidb/issues/52742) @[winoros](https://github.com/winoros)
+    - 将具有 nulleq 条件的笛卡尔积 Semi Join 转换为具有相等条件的 Semi Join [#57583](https://github.com/pingcap/tidb/issues/57583) @[hawkingrei](https://github.com/hawkingrei)
+    - 将统计信息内存缓存的默认阈值调整为总内存的 20% [#58014](https://github.com/pingcap/tidb/issues/58014) @[hawkingrei](https://github.com/hawkingrei)
+
++ TiKV <!--tw@Oreoxmt: 1 note-->
+
+    - 增加对非法 `max_ts` 更新的检测机制 [#17916](https://github.com/tikv/tikv/issues/17916) @[ekexium](https://github.com/ekexium)
+
++ TiFlash
+
+    - (dup): release-7.5.5.md > 改进提升> TiFlash - 优化在存算分离架构下，读节点从 S3 下载文件异常时的重试策略 [#9695](https://github.com/pingcap/tiflash/issues/9695) @[JinheLin](https://github.com/JinheLin)
+
++ Tools
+
+    + TiCDC <!--tw@qiancai: 1 note-->
+
+        - 提前过滤掉未被 TiCDC 订阅的事件，以避免不必要的资源消耗 [#17877](https://github.com/tikv/tikv/issues/17877) @[hicqu](https://github.com/hicqu)
```

Comment:
```suggestion
        - - Filter out events that are not subscribed to by TiCDC in advance to avoid unnecessary resource consumption [#17877](https://github.com/tikv/tikv/issues/17877) @[hicqu](https://github.com/hicqu)
```